src/soc/amd/common/boot/src/lib.rs

```diff
@@ -0,0 +1,222 @@
+#![feature(llvm_asm)]
+#![feature(lang_items, start)]
+#![no_std]
+#![no_main]
+#![feature(global_asm)]
+
+use arch::bzimage::BzImage;
+use arch::ioport::IOPort;
+use model::Driver;
+use uart::i8250::I8250;
+mod acpi;
+use acpi::setup_acpi_tables;
+extern crate heapless; // v0.4.x
+use heapless::consts::*;
+use heapless::Vec;
+
+use core::ptr;
+// Until we are done hacking on this, use our private copy.
```

Comment:
Thiis comment probably no longer applies

Reply:
Agreed--will/did remove in phase 2, in a new pull requested.